### DISCUSSION TITLE
Possibility of using REAP framework for container migration

### DISCUSSION THREAD

**Discussion by niravnshah** [vhive-serverless/vHive#497] [Ideas]
Hi, 

I would like to start a discussion around the possibility of using REAP framework in container migration.
Has there been any thoughts/research done in that area? If yes, it would be nice to know the outcomes.
If no, any thoughts on if and how REAP framework can benefit container migration.

Thanks,
Nirav.

**Comment by niravnshah** [comment]
Hi @ustiugov , any thoughts on this?

**Reply by ustiugov** [reply]
Hi Nirav, it is possible but "migration" in the serverless is equivalent to cold starts with a remote snapshot storage. This would be possible one #465 is merged

**Comment by niravnshah** [comment]
Thanks @ustiugov , I will keep an eye on that. 👍
